auslib/admin/views/releases.py

```diff
@@ -241,8 +241,12 @@ def _put(self, release, changed_by, transaction):
                 msg = "Couldn't update release: %s" % e
                 cef_event("Bad input", CEF_WARN, errors=msg)
                 return Response(status=400, response=json.dumps({"data": e.args}))
+            except OutdatedDataError as e:
+                msg = "Couldn't update release: %s" % e
+                cef_event("Bad input", CEF_WARN, errors=msg)
+                return Response(status=400, response=json.dumps({"data": e.args}))
             data_version += 1
-            return Response(json.dumps(dict(new_data_version=data_version)), status=200)
+            return Response(json.dumps(dict(new_data_version=data_version)), status=200)            
```

Comment:
yeah I can do that.so I'm making the code more DRY by adding the exception handlers in base class itself. anything else I can try other than this ?